lms/static/js/student_profile/views/learner_profile_factory.js

```diff
@@ -40,7 +40,9 @@
                 required: true,
                 editable: 'always',
                 showMessages: false,
-                title: gettext('edX learners can see my:'),
+                title: interpolate_text(
+                    gettext('{platform_name} learners can see my:'), {platform_name: options.platform_name}
```

Comment:
Ah, I see now, thanks for clarifying, @andy-armstrong and @srpearce. This PR is merged, so if we want to make a quick tweak to the copy, then we should open a separate PR.